docs/pages/management/guides/docker.mdx

```diff
@@ -149,10 +151,104 @@ The Web UI will be available at the displayed URL.
 
 (!docs/pages/includes/insecure-certificate.mdx!)
 
-## Step 4/4. tsh into your Teleport container
+## Step 4/4. Try Server Access
 
-After you have finished creating your user, open a second terminal and issue the
-command, which will log in to your Teleport cluster via the Proxy Service at
+This docker installation includes just the Teleport Proxy and Auth Service.  You can try
+accessing a server through Teleport by installing the Teleport SSH Service
+on a Ubuntu docker container. Here are the steps.
```

Comment:
```suggestion
on an Ubuntu Docker container. Here are the steps.
```

Reply:
Also, does this have to be Ubuntu, or are we using Ubuntu to demonstrate? This kind of makes it sound like the former, even though we know it's the latter.

Reply:
Ubuntu is just a example and matches to the cloud example.